src/ActionList/Item.tsx

```diff
@@ -151,9 +151,13 @@ const getItemVariant = (variant = 'default', disabled?: boolean) => {
   }
 }
 
-const StyledItemContent = styled.div`
+const StyledItemContent = styled.div<{
+  descriptionVariant: ItemProps['descriptionVariant']
+}>`
+  align-items: baseline;
   display: flex;
   min-width: 0;
+  flex-direction: ${({descriptionVariant}) => (descriptionVariant === 'inline' ? 'row' : 'column')};
```

Comment:
Want to try changing this to a css variable?  I do think it will lead to much better performance if we do it consistently throughout the components, and this seems like a good place to start.  Let me know if you want to pair on it

Reply:
👍 Migrated in https://github.com/primer/components/pull/1320/commits/73eed06a2f81cd18e657f0a0c15f62cbde924555